x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_jsbase/ml/anomalous_error_rate_for_user_agents_jsbase.json

```diff
@@ -0,0 +1,40 @@
+{
+  "job_type": "anomaly_detector",
+  "groups": [
+    "apm"
+  ],
+  "description": "APM JSBase: Detects user agents that are encountering errors at an above normal rate. This can help detect browser compatibility issues.",
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "high_non_zero_count partitionfield=\"user_agent.name\"",
```

Comment:
`"description": "high error rate for user agent"`

Reply:
https://github.com/elastic/kibana/pull/53792/commits/ebe6353130f170600b92e45be911f70264e4f4be